plugin/src/main/java/org/opensearch/ml/sdkclient/SdkClientFactory.java

```diff
@@ -92,16 +88,20 @@ public static SdkClient createSdkClient(Client client, NamedXContentRegistry xCo
                     throw new OpenSearchException("AWS Opensearch Service client requires a metadata endpoint and region.");
                 }
                 log.info("Using remote AWS Opensearch Service cluster as metadata store");
-                return new SdkClient(new RemoteClusterIndicesClient(createAwsOpenSearchServiceClient(remoteMetadataEndpoint, region)));
+                return new SdkClient(
+                    new RemoteClusterIndicesClient(createAwsOpenSearchServiceClient(remoteMetadataEndpoint, region, "es"))
+                );
             case SdkClientSettings.AWS_DYNAMO_DB:
                 if (Strings.isBlank(remoteMetadataEndpoint) || Strings.isBlank(region)) {
-                    throw new OpenSearchException("AWS Opensearch Service client requires a metadata endpoint and region.");
+                    throw new OpenSearchException(
+                        "AWS DynamoDB and Opensearch Serverless Service client requires a metadata endpoint and region."
```

Comment:
Do we need to mention about `Serverless`?

Reply:
"Need"? Probably not. But the "aoss" string in the code on line 104 means exactly that.

Should we make the signing service a configurable setting and vary the message based on that?